source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -182,6 +182,41 @@ def body_incoming_wrench(env: ManagerBasedEnv, asset_cfg: SceneEntityCfg) -> tor
     return link_incoming_forces.view(env.num_envs, -1)
 
 
+def grab_images(
+    env: ManagerBasedEnv,
+    sensor_cfg: SceneEntityCfg = SceneEntityCfg("tiled_camera"),
+    data_type: str = "rgb",
+    convert_perspective_to_orthogonal: bool = True,
+    normalize: bool = True,
+) -> torch.Tensor:
+    """Grab all of the images of a specific datatype produced by a specific camera at the last timestep.
+
+    Args:
+        env: The environment the cameras are placed within.
+        sensor_cfg: The desired sensor to take . Defaults to SceneEntityCfg("tiled_camera").
+        data_type: The data type to pull from the desired camera. Defaults to "rgb".
+        convert_perspective_to_orthogonal: Whether to convert perspective
+            depth images to orthogonal depth images. Defaults to True.
+        normalize: Set to true normalize images. Highly recommended for training performance.
```

Comment:
NIT: I would hold back on the highly recommended for training performance
```suggestion
        normalize: Set to True to normalize images.
```